pybv/io.py

```diff
@@ -156,8 +156,12 @@ def _write_vmrk_file(vmrk_fname, eeg_fname, events):
         if events is None or len(events) == 0:
             return
 
-        # Handle events
+        # Handle events: We write all of them as "Stimulus" events for now.
+        # This is a string staring with "S" and followed by an integer of
+        # minimum length 3, padded with "space" if the integer is < length 3.
+        # For example "S  1", "S 23", "S345"
         twidth = int(np.ceil(np.log10(np.max(events[:, 1]))))
+        twidth = twidth if twidth > 3 else 3
```

Comment:
what are the reasons that it would be greater than 3?

Reply:
I think there are none (waiting for a reply by BrainProducts, whether this would even be supported)

but until we know for sure, I think it's better so only fix what is broken :-)

Reply:
makes sense to me - maybe we should add in a comment saying as much? I'm always nervous about adding in these kinds of lines where it's not explicit why "3" is the right number here. Right now we know this is the case, but I imagine a future where we look at this line of code and are like "why the hell did we cap it to 3?!"

Reply:
sounds good :-) fixed in https://github.com/bids-standard/pybv/pull/26/commits/1baee691aa9379b550cf4fd6679471a3b5dae14b